src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -42,6 +45,32 @@ public Errors validate(OverseasEntitySubmissionDto overseasEntitySubmissionDto,
                 errors,
                 loggingContext);
 
+        ownersAndOfficersDataBlockValidator.validateOwnersAndOfficersAgainstStatement(overseasEntitySubmissionDto, errors, loggingContext);
+        return errors;
+    }
+
+    public Errors validatePartial(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+
+        var presenterDto = overseasEntitySubmissionDto.getPresenter();
+        if (Objects.nonNull(presenterDto)) {
+            presenterDtoValidator.validate(presenterDto, errors, loggingContext);
+        }
+
+        var dueDiligenceDto = overseasEntitySubmissionDto.getDueDiligence();
+        var overseasEntityDueDiligenceDto = overseasEntitySubmissionDto.getOverseasEntityDueDiligence();
+        if (Objects.nonNull(dueDiligenceDto) || Objects.nonNull(overseasEntityDueDiligenceDto)) {
+            dueDiligenceDataBlockValidator.validateDueDiligenceFields(
+                    dueDiligenceDto,
+                    overseasEntityDueDiligenceDto,
+                    errors,
+                    loggingContext);
+        }
+
+        var entityDto = overseasEntitySubmissionDto.getEntity();
```

Comment:
the entity block should be moved further up the method so we validate the blocks in the screen order, it will also help with the ordering of error messages, ie. it would be better to have entity errors earlier in the list than due diligence.

Reply:
Entity is after due diligence.

Reply:
ok, i had it in my head it was earlier.